rts/at/util/src/main/java/org/jboss/jbossts/star/util/logging/at18NLogger.java

```diff
@@ -0,0 +1,124 @@
+package org.jboss.jbossts.star.util.logging;
+
+import static org.jboss.logging.Logger.Level.ERROR;
+import static org.jboss.logging.Logger.Level.WARN;
+
+import javax.transaction.xa.XAException;
+
+//import javax.json.JsonObject;
+
+import org.jboss.logging.annotations.Cause;
+import org.jboss.logging.annotations.LogMessage;
+import org.jboss.logging.annotations.Message;
+import org.jboss.logging.annotations.MessageLogger;
+
+/**
+ * at18N log messages for the rest-at module.
+ */
+@MessageLogger(projectCode = "AT")
+public interface at18NLogger {
+	
+
+    /*
+        Message IDs are unique and non-recyclable.
+        Don't change the purpose of existing messages.
+          (tweak the message text or params for clarification if you like).
+        Allocate new messages by following instructions at the bottom of the file.
+     */
+	@LogMessage(level = ERROR)
+    @Message(id = 27001, value = "REST-AT transaction is required for this request. '%s'")
+    void error_rest_at_transactionrequired(int statusCode, @Cause Throwable t);
```

Comment:
please, remove the `_rest_at` prefix at all the method names in this class. Any use of them is prefixed with the `at18NLogger` and it's obvious they're part of this REST AT logging class. Having every method prefixed this  way just brings naming duplicity.

Reply:
I thought about that too and agree that it makes sense.